<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>256</width>
    <height>256</height>
   </size>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::GroupedDragging|QMainWindow::VerticalTabs</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;a href=&quot;https://github.com/Advik-B/CMPDL&quot;&gt;&lt;span style=&quot; font-size:16pt; text-decoration: underline; color:#000000;&quot;&gt;C.M.P.D.L&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#000000;&quot;&gt;Curseforge ModPack DownLoader&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="full_logs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>471</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="download_tab">
     <attribute name="title">
      <string>Download from modpack</string>
     </attribute>
     <widget class="QLineEdit" name="modpack_pth">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Path to modpack file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browse_modpack">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="download_options">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>441</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Download Options</string>
      </property>
      <widget class="QCheckBox" name="optional_mods">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Download optional mods</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="keep_config">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Keep config (if any)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="download_pth">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Download path</string>
       </property>
      </widget>
      <widget class="QPushButton" name="download_pth_browse">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>20</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>441</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Progress</string>
      </property>
      <widget class="QProgressBar" name="per_mod_progress">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="format">
        <string>%p%</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>401</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Overall Progress</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progress">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>421</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Per-mod download progress</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="logs_area">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>451</width>
        <height>191</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTabWidget" name="mods_info_tab">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>9</y>
         <width>441</width>
         <height>181</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="download_info_tab">
        <attribute name="title">
         <string>Download info</string>
        </attribute>
        <widget class="QListWidget" name="DownloadList">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>10</y>
           <width>421</width>
           <height>131</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="logs_and_debug_tab">
        <attribute name="title">
         <string>Logs and Debug</string>
        </attribute>
        <widget class="QPlainTextEdit" name="logs_box">
         <property name="geometry">
          <rect>
           <x>3</x>
           <y>16</y>
           <width>411</width>
           <height>131</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="backgroundVisible">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="start_download_btn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>590</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Start Download</string>
      </property>
     </widget>
     <widget class="QPushButton" name="copy_logs_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>590</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Copy Logs</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="create_tab">
     <attribute name="title">
      <string>Create your own modpack</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>351</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Comming Soon&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
